src/components/pages/bc-alby-go.ts

```diff
@@ -99,8 +99,7 @@ ${classes['text-brand-mixed']} ${classes.interactive} font-semibold text-xs"
 
     return html`
       <!-- add margin only on dark mode because on dark mode the qr has a white border -->
-
-      <canvas id="qr"></canvas>
+      <canvas id="qr" class="dark:bg-[#fff] dark:p-4"></canvas>
```

Comment:
could we use `bg-white` as it's a standard tailwind class

Reply:
ok, i'll change all three.